stemcell_builder/stages/prepare_rawdisk_image_stemcell/apply.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2009-2012 VMware, Inc.
+
+set -e
+
+base_dir=$(readlink -nf $(dirname $0)/../..)
+source $base_dir/lib/prelude_apply.bash
+
+pushd $work
+rm -f disk.raw
+ln ${stemcell_image_name} disk.raw
+tar zcf stemcell/image disk.raw
+popd
```

Comment:
We discovered that GCE requires that the uploaded machine image tarball contain a `disk.raw` file: https://cloud.google.com/compute/docs/tutorials/building-images#packaging.